evaluators/text/HammingDistanceEvaluator/__init__.py

```diff
@@ -0,0 +1,22 @@
+from jina.executors.evaluators.text import BaseTextEvaluator
+
+class HammingDistanceEvaluator(BaseTextEvaluator):
+    """A:class:`HammingDistanceEvaluator` Gives the Hamming distance between result and groundtruth string..
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    @property
+    def metric(self):
+        return 'HammingDistance'
+
+    def evaluate(self, actual: str, desired: str):
+        if len(actual) != len(desired):
+            raise ValueError("Undefined for sequences of unequal length.")
```

Comment:
Hi, thanks for your contributions!
We use single quotations, could you change those?

Reply:
of course!